### PR TITLE
Fix the issue cause build failure for Xcode 10.2

### PR DESCRIPTION
**Pull request by dreampiggy**
Fix the wrong relative path of SDWebImage.xcodeproj
Fix the wrong xcconfig using ENABLE_BITCODE on debug configuration

### New Pull Request Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar pull request in the [project](https://github.com/rs/SDWebImage/pulls) and found none

* [x] I have updated this branch with the latest master to avoid conflicts (via merge from master or rebase)
* [x] I have added the required tests to prove the fix/feature I am adding
* [x] I have updated the documentation (if necessary)
* [x] I have run the tests and they pass
* [x] I have run the lint and it passes (`pod lib lint`)

This merge request fixes / refers to the following issues: ...

### Pull Request Description

Xcode 10.2 today is official released.

Current 5.x branch can not build the SDWebimage.xcworkspace using Xcode 10.2. Because of the following errors:

### The <WatchKit/WatchKit.h> import error (Could not build module 'WatchKit')

This is because we enable the Bitcode on even the Debug configuration for WatchKit Demo. However, this should be turned on only the Release configuration. And this cause the error below:

image

### The `Multiple commands produce '<Touch /Users/lizhuoli/Library/Developer/Xcode/DerivedData/` error

This is because we reference the `SDWebImage.xcodeproj` in the workspace, but the path use a relative path and cause Xcode confused.

`location = "group:Examples/../SDWebImage.xcodeproj">`

Chaning this into the root path works without issue.

`location = "group:SDWebImage.xcodeproj">`

### CODE REVIEW

**Comment by dreampiggy** [comment]
Tested with these fix, and Xcode 10.2(10E125) can build successfully.

image

This changes, only effect the Demo and Test project, does not effect the framework itself. So it's safe to merge in.

**Comment by dreampiggy** [comment]
Also, fix the warning of the `English.lproj` to `en.lproj`, which is introduced during Xcode 9 and the old style is been deprecated.

image

File: SDWebImage.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -923,10 +923,11 @@
 			};
 			buildConfigurationList = 53922D69148C55810056699D /* Build configuration list for PBXProject "SDWebImage" */;
 			compatibilityVersion = "Xcode 3.2";
-			developmentRegion = English;
+			developmentRegion = en;
 			hasScannedForEncodings = 0;
 			knownRegions = (
 				en,
+				Base,
```
Comment (kinarob) [inline SDWebImage.xcodeproj/project.pbxproj]: Xcode automatic fix behavior with `Base` and `en`

Comment (dreampiggy) [reply SDWebImage.xcodeproj/project.pbxproj]: Need this be changed ? Or keep ?

**Comment by dreampiggy** [comment]
> Run failed in `Release` environment from you branch, 😂😂

Fixed by c292fcc

**Comment by dreampiggy** [comment]
For Xcode 10.2, seems the WatchOS/WatchExtension target, should not using `ENABLE_BITCODE` = YES. The build setting is totally removed from the Xcode GUI of build setting.

So, instead of writing complicated if else in the xcconfig, simply remove the `ENABLE_BITCODE` and fallback the default config. Which solve the problem.

Bitcode support:

+ macOS: NO, no build setting
+ iOS: Default YES, have build setting
+ tvOS: YES, no build setting
+ watchOS: YES, no build setting

File: SDWebImage.xcodeproj/project.pbxproj (hunk 1 above)
Comment (kinarob) [reply SDWebImage.xcodeproj/project.pbxproj]: Keep

**Comment by Insofan** [comment]
Upgrade to  10.2 (10E125), I couldn't build. I encountered "Could not build module 'WatchKit' " and "GNU-style ..." errors.

**Comment by Insofan** [comment]
Under 10.2 (10E125). Sometimes I checkout other branch then go back, XCode build failure, almost 30 percent. I cd to Example directory `pod update` it will fix, almost 90 percent. If it still build failure, I have to replay these operations.

**Comment by dreampiggy** [comment]
@Insofan You can temporaily change to the Legacy build System.

Or we have to wait CocoaPods 1.7.1 to solve the problem. This because of CocoaPods generate two `Pods.xcodeproj` into the workspace, and cause the issue.

**Pull request merged**